docs/docs/migrating-from-v1-to-v2.md

```diff
@@ -843,7 +843,7 @@ In most cases you won't have to do anything to be v2 compatible, however there a
 
 ```json
 "peerDependencies": {
-  "gatsby": ">=1"
+  "gatsby": ">=2"
```

Comment:
I believe this should read 2 instead of 1 if I'm reading this section correctly, which is explaining how plugin authors can update for v2 support.

![image](https://user-images.githubusercontent.com/2344137/45073256-4fc82d00-b093-11e8-8d78-fa9597ef9f44.png)

Reply:
If the plugin works for Gatsby v1 and v2 then `>=1` is fine.  A lot of plugins specify `1` as the supported version. 

Maybe this docblock could be changed from `json` to `diff` and updated to show the change?

```
-  "gatsby": "1"
+  "gatsby": ">=1"
```